src/linter.js

```diff
@@ -263,8 +263,12 @@ export default class Linter {
       _log.info('Retrieving metadata from manifest.json');
       const json = await this.io.getFileAsString(constants.MANIFEST_JSON);
       const manifestParser = new ManifestJSONParser(json, this.collector, {
-        selfHosted: this.config.selfHosted,
         io: this.io,
+        selfHosted: this.config.selfHosted,
```

Comment:
Should that one go into `validatorOptions` too? Not sure what `validatorOptions` is yet, though.

Reply:
@willdurand I was wondering if "that one" in this comment does refer to the consts from comment https://github.com/mozilla/addons-linter/pull/3633#discussion_r624936948, if that is the case, does https://github.com/mozilla/addons-linter/pull/3633#discussion_r625039218 provide you enough context to see why those consts are not explicitly used inside validatorOptions?

basically, is it clear enough why `MANIFEST_VERSION_MAX` and `getDefaultConfigValue('max_manifest_version')` are not the same thing? otherwise it may be worth to mention that more explicitly some where:
- `MANIFEST_VERSION_MAX` represents the `max_manifest_version` we would use for schema entries that do now have a max_manifest_version on their own (which means "we consider this API still valid at least up to this manifest version)
- `getDefaultConfigValue('max_manifest_version')` represents the maximum manifest_version that we will consider valid while validating an extension (by default is currently set to 2, and only extensions with `manifest_version: 2` would be considered valid for submission to AMO)

Reply:
It's perfectly clear.

"that one" was referring to `selfHosted` actually but I found later that it was passed to `validator` and I suggested to rename `validatorOptions` to `schemaValidatorOptions`. It's a bit more verbose but it is also more explicit.

(I should maybe update some comments before submitting my reviews, I kinda want to give how I approached the review but I get that it can be confusing)

Reply:
ah, I see now, and yeah, and my willingness to rename `validatorOptions` into `schemaValidatorOptions` just got increased as a side effect :-)

> (I should maybe update some comments before submitting my reviews, I kinda want to give how I approached the review but I get that it can be confusing)

don't worry, I actually find that useful because it provides me additional hints about what wasn't immediately clear while reading the changes gradually without having the full picture yet (and push me to think into ways to make those points more immediately clear) and so I don't actually mind that.